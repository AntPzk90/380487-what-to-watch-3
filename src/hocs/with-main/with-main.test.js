import React from 'react';
import renderer from 'react-test-renderer';
import withMain from './with-main.jsx';


it(`SnapshotTest withMain`, () => {

  const tree = renderer
  .create(
      <withMain
        onShowMoreBtnClick={() => {}}
        count={8}
        isShowBtn={false}
      />
  )
  .toJSON();

  expect(tree).toMatchSnapshot();
});
