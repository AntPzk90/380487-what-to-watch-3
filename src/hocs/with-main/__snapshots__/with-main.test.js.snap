// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnapshotTest withMain 1`] = `
<withMain
  count={8}
  isShowBtn={false}
  onShowMoreBtnClick={[Function]}
/>
`;
